app/controllers/assignments_controller.rb

```diff
@@ -208,16 +208,14 @@ def index
       @section = current_user.section
       # get results for assignments for the current user
       @a_id_results = Hash.new()
-      @assignments.each do |a|
-        if current_user.has_accepted_grouping_for?(a)
-          grouping = current_user.accepted_grouping_for(a)
-          if grouping.has_submission?
-            submission = grouping.current_submission_used
-            if submission.has_remark? && submission.remark_result.released_to_students
-              @a_id_results[a.id] = submission.remark_result
-            elsif submission.has_result? && submission.get_original_result.released_to_students
-              @a_id_results[a.id] = submission.get_original_result
-            end
+      accepted_groupings = current_user.accepted_groupings.includes(:assignment, :current_submission_used)
+      accepted_groupings.each do |grouping|
+        if !grouping.assignment.id_hidden && grouping.has_submission?
```

Comment:
There's a typo on this line with the assignment attribute. (The fact that this was not caught means you didn't test the code in all possible cases here; not a big deal, but please keep in mind for next time.)

Reply:
(I realize I made the original typo, but still good to double-check.)

Reply:
Yes, sorry I should have caught this! I realized I deployed the wrong version when I tested.